components/core/cmake/Modules/FindMariaDBClient.cmake

```diff
@@ -20,6 +20,17 @@ include(cmake/Modules/FindLibraryDependencies.cmake)
 find_package(PkgConfig)
 pkg_check_modules(mariadbclient_PKGCONF QUIET "lib${mariadbclient_LIBNAME}")
 
+if(NOT mariadbclient_PKGCONF_FOUND AND APPLE)
+    execute_process(COMMAND brew --prefix mariadb-connector-c OUTPUT_VARIABLE mariadb_MACOS_PREFIX)
```

Comment:
Following the comments at the top of the file, this should be `mariadbclient_MACOS_PREFIX`.